pdns/opensslsigners.cc

```diff
@@ -211,6 +211,19 @@ class OpenSSLRSADNSCryptoKeyEngine : public DNSCryptoKeyEngine
 
 void OpenSSLRSADNSCryptoKeyEngine::create(unsigned int bits)
 {
+  if (d_algorithm == 5 && (bits < 512 || bits > 4096)) {
```

Comment:
Don't we have named constants for the algorithm?

Reply:
Indeed we do. Not sure why half of the code is using the numbers directly.